src/handlers.pxi

```diff
@@ -45,9 +45,11 @@ cdef void fuse_init (void *userdata, fuse_conn_info *conn):
         conn.capable & FUSE_CAP_POSIX_ACL):
         conn.want |= FUSE_CAP_POSIX_ACL
 
-    # Blocking rather than async, in case we decide to let the
-    # init hander modify `conn` in the future.
-    operations.init()
+    cdef object conn_obj = ConnInfo(max_read=conn.max_read)
+    # Blocking rather than async, to let the init hander modify `conn_obj`.
```

Comment:
This comment doesn't make sense. async functions can modify their parameters too. Just omit it entirely, having it sync is fine.